src/Base/Bitmask.h

```diff
@@ -128,5 +129,6 @@ class Flags {
     }
 };
 }
+// clang-format on
 
```

Comment:
Same question. Just marking the block in case the answer is different.

Reply:
Same answer.